src/components/ThirdPartyCollectionDetailPage/ThirdPartyCollectionDetailPage.tsx

```diff
@@ -39,14 +44,15 @@ export default class ThirdPartyCollectionDetailPage extends React.PureComponent<
   }
 
   hasAccess() {
-    const { wallet, collection } = this.props
-    return collection !== null && canSeeCollection(collection, wallet.address)
+    const { wallet, collection, thirdParty } = this.props
+    return collection && thirdParty && isUserManagerOfThirdParty(wallet.address, thirdParty)
```

Comment:
This is enough to verify that the user has access to the collection. What do you think about this @nicosantangelo ?

Reply:
Yup I think it's ok!